src/sentry/receivers/releases.py

```diff
@@ -53,6 +58,56 @@ def resolve_group_resolutions(instance, created, **kwargs):
     clear_expired_resolutions.delay(release_id=instance.id)
 
 
+def resolved_in_commit(instance, created, **kwargs):
+    # TODO(dcramer): we probably should support an updated message
+    if not created:
+        return
+
+    match = re.search(r'\bFixes ([A-Za-z0-9_-]+-[A-Z0-9]+)\b')
```

Comment:
Should probably be set to `(?i)` to make it case insensitive. People often write "This fixes". Also maybe `\s+` instead of a space?